cmd/worker/main.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"emperror.dev/emperror"
 	"emperror.dev/errors"
+	"github.com/banzaicloud/pipeline/internal/clusterfeature/features/securityscan"
```

Comment:
Please configure your IDE to automatically fix go imports violations. (Wonder why golangci didn't catch this one)

Reply:
don't really know what's the matter here, i ran goimports, gofmt - nothing happens :)

Reply:
Please rebase your branch after #2336 gets merged.

Reply:
thanx, did that ...